enable_safety_check: true
Fext_dead_zone: [0.5; 0.5; 0.5; 0.35; 0.35; 0.35]
vel_limit: [0.7; 0.7; 0.7; 1.2; 1.2; 1.2]
wrench_limit: [20; 20; 20; 25; 25; 150]
